catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -128,10 +129,16 @@ public void run() {
           comparator = new DistanceResultComparator(sortOrder);
         } else if (Result.RELEVANCE.equals(sortType)) {
           comparator = new RelevanceResultComparator(sortOrder);
+        } else {
+          comparator =
+              Comparator.comparing(
+                  r -> getAttributeValue((Result) r, sortingProp.getPropertyName()),
```

Comment:
Do date properties come back as date objects?  What if the user tries to sort on created date?

Reply:
This will work as `Date` implements `Comparable` https://docs.oracle.com/javase/8/docs/api/java/util/Date.html

Reply:
cool, just wasn't sure they were returned as Date objects